apollo-router/src/configuration/metrics.rs

```diff
@@ -494,6 +496,18 @@ impl InstrumentData {
         );
     }
 
+    pub(crate) fn populate_legacy_fragment_usage(&mut self, configuration: &Configuration) {
+        // Fragment generation takes precedence over fragment reuse. Only report when fragment reuse is *actually active*.
+        if configuration.supergraph.reuse_query_fragments == Some(true)
+            && !configuration.supergraph.generate_query_fragments
+        {
+            self.data.insert(
+                "apollo.router.config.experimental_reuse_query_fragments".to_string(),
```

Comment:
```suggestion
                "apollo.router.config.reuse_query_fragments".to_string(),
```
No need to have experimental in the metric name